engine/common/cloud_auth_api/client.py

```diff
@@ -73,7 +78,10 @@ def request_signed_token(
             url,
             headers=headers,
             json={
-                "claims": claims,
+                "claims": {
+                    "sub": f"user:{user.user_id}",
```

Comment:
@twu @paulcoghlan wdyt about this?

Reply:
Is the email address not needed in addition to this?

Reply:
TBH, I don't know - I'll do an analysis of the gateway [logs](https://ops.grafana-ops.net/explore?schemaVersion=1&panes=%7B%22cge%22:%7B%22datasource%22:%22000000193%22,%22queries%22:%5B%7B%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22000000193%22%7D,%22editorMode%22:%22builder%22,%22expr%22:%22%7Bjob%3D%5C%22incident%2Fcortex-gw%5C%22%7D%20%7C%3D%20%60POST%20%2Fincident%2Fapi%60%20%7C%20pattern%20%60%3C_%3E%20msg%3D%5C%22%3Cmethod%3E%20%3Cpath%3E%20%28%3Cstatus%3E%29%20%3Clatency%3E%20%3C_%3E%5C%22%60%20%7C%20logfmt%22,%22queryType%22:%22range%22,%22refId%22:%22A%22%7D%5D,%22range%22:%7B%22from%22:%22now-5m%22,%22to%22:%22now%22%7D%7D%7D&orgId=1) I can figure out what we're using

Reply:
Yes, it looks like `email:` is the most common identifier we use. @joeyorlando - maybe you could add it as `email:<email-address>`?

Reply:
thanks for looking into that @paulcoghlan 🙌 

updated this in https://github.com/grafana/oncall/pull/4194/commits/5a639d557791265c9f7f7702072b4be40bccda8b